### ISSUE TITLE
Add a 'Limit' option for List Fields

### ISSUE THREAD

**Issue by ItsMeAra** [getgrav/grav-plugin-admin#956]
When using [The List Field](https://learn.getgrav.org/forms/blueprints/fields-available#the-list-field) it would be nice to be able to set a limit on the number of Items that can be added through the admin. For example, I tried `limit: 1` but didn't do anything, still allows unlimited items. 

```yaml
header.hero_button:
  name: button
  type: list
  label: Button
  limit: 1  # <~ this would be great!
  fields:
    .text:
      type: text
      label: Button Text
      validate:
        required: true
    .url:
      type: text
      label: Button URL
      validate:
        required: true
    .theme:
      type: select
      label: Button Color
      options:
          1: 'Blue'
          0: 'Green'
```

Discovered in Slack chat that this is not a current feature. Would be nice to add. 😄

**Comment by dimitrilongo** [comment]
Same here https://github.com/getgrav/grav-plugin-admin/issues/720

**Comment by w00fz** [comment]
Fixed in #1113, thanks @david-szabo97
